docs/user-manual.rst

```diff
@@ -915,6 +915,108 @@ will compute a boolean that tells you whether the field "000000" passes
 a Chi-square test for two degrees of freedom with significance level
 0.05.
 
+
+Fuzzy logic
+-----------
+
+Flatline provides some functions to work with Fuzzy logic datasets.
+Fuzzy logic is a form of many-valued logic in which the truth values
+of variables may be any real number between 0 and 1 inclusive. This
+kind of functions are called **t-norms** and **t-conorms**.
+
+You can find more information about t-norms and t-conorms in the
+following links:
+
+- `Wikipedia: t-norms and t-conorms <https://en.wikipedia.org/wiki/T-norm>`__
+- `Wikipedia: Construction of t-norms <https://en.wikipedia.org/wiki/Construction_of_t-norms>`__
+
+All the norms need, at least, two arguments. The 2 fields where
+t-norms will be applied to. You can specify them with an string
+representing their field id or its field name. However, you can also
+use this norms with two real numbers as arguments, and the norms will
+be applied to these numbers:
```

Comment:
I would shorten this full paragraph.
Perhaps: "All the norms are computed from two numeric values that can be 
specified either by referencing a numeric input field (giving its name or id)
or by any valid flatline numeric expression.  For instance:"